exercise_data/python_requirements.txt

```diff
@@ -0,0 +1,7 @@
+numpy==1.24.1
```

Comment:
Can you put this file as comments in the python conversion file?

Reply:
I can, but the usual way to go is to `pip install -r requirements.txt`, so this way wold be more convenient

Reply:
well, okay, but the location of the file missleads that it regards the whole exercise data not just data 18. Maybe make a folder `18_generatio` and put there the python and requirement files?